DigitalLearningSolutions.Web/Styles/shared/components/cancelLink.scss

```diff
@@ -0,0 +1,13 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.nhsuk-back-link__link {
+  padding-left: 24px;
+}
+
+.nhsuk-icon__close {
+  height: 24px;
+  left: -2px; /* 2 */
+  position: absolute;
+  top: -1px; /* 3 */

```

Comment:
What do the comments mean?

Reply:
I'd just copied the style from the NHS css for the back link, it has a few comments to describe what they do (basically alignment of the svg)